src/EditorFeatures/Core/SymbolMonikers/SymbolMoniker.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System;
+
+namespace Microsoft.CodeAnalysis.Editor.SymbolMonikers
+{
+    internal class SymbolMoniker
```

Comment:
Moved from Jason's LSIF pr

Reply:
Looks like we removed the internal/external scoping? You won't need it, but I'll have to add it back?

Reply:
yes :)

Reply:
Not a problem, just wanted to confirm intent.